modules/ROOT/pages/change-runtime-version.adoc

```diff
@@ -0,0 +1,19 @@
+= Change the Version of Mule Runtime that a Mule App Uses
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can change an existing Mule application to use a newer version of Mule Runtime. You might want to do this if the newer version includes updates or new features that you want your application to take advantage of.
+
+== Procedure
+. Open the project.
+. Click the gear icon that is in the lower-left corner of the screen.
+. In the Project Settings dialog, select the version that you want to switch to. The dialog lists a recommended version, which is the most recent version that has been tested for compatibility with Design Center.
+
+== Result
+
+Changing the version of Mule runtime might require you to update the connectors that are in the application's flows. Only after updating the connectors would be able to redeploy the application.
```

Comment:
Mule runtime engine

This applies throughout so I won't keep commenting.

Reply:
? "Only after updating the connectors would be able to redeploy the application."

You can redeploy the application only after updating the connectors. 

??